### ISSUE TITLE
Change power law reference frequency to 1 mHz

### ISSUE THREAD

**Issue by criswellalexander** [sharanbngr/blip#128]
Issue: the current default reference frequency for power law spectral models in BLIP is 25 Hz, which is a relic from ground-based interferometers. The more common choice for LISA is 1 mHz.

Steps:
- Set the default fref in the argparser to 1 mHz
- Remove fref as a user-definable variable
- Recompute all prior bounds for amplitude (Omega_ref/Omega0), adjusting for the new reference frequency and assuming a 2/3 slope of the standard power law. (Note that the current normalization for the broken/truncated power laws is calculated for the equivalent unbroken/untruncated power law at 25Hz, so the process will be the same for all power-law-based spectral models.)
- Test each major spectral model with the updated priors, check that the result is consistent with the same analysis without your changes (modulo the normalization)

**Comment by pushkar-hue** [comment]
I would like to work on this issue can you tell me where can I start? Where do i find the file which contains argparser?

**Comment by criswellalexander** [comment]
Hi -- the argparser is in run_blip.py. Do you have prior experience working with Bayesian stats/gravitational wave background spectra? If not, there may be some more code-focused issues that might be a better fit. (I'm asking b/c I don't see any related work in your git profile)

--AWC

**Comment by pushkar-hue** [comment]
No, you're right, I thought I'd learn as I do. Can you guide me is there any other issues I can work on that are more of a fit for me.
